docs/src/configuration/services/_index.md

```diff
@@ -1,177 +1,182 @@
 
-All services offer one or more `endpoints`.  An endpoint is simply a named set of credentials that can be used to give access to other applications and services in your project to that service.  Only some services support multiple user-defined endpoints.  If you do not specify one then one will be created with a standard defined name, generally the name of the service type (e.g., `mysql` or `solr`).  An application container, defined by a `.platform.app.yaml` file, always exposes an endpoint named `http` to allow the [router](/configuration/routes/_index.md) to forward requests to it.
+## Service timezones
 
-When defining relationships in a configuration file you will always address a service as `<servicename>`:`<endpoint>`.  See the appropriate service page for details on how to configure multiple endpoints for each service that supports it.
+All services have their system timezone set to UTC by default.
+For some services, you can change the timezone for the running service
+(this doesn't affect the container itself and so logs are still in UTC).
 
-## Connecting to a service
+* [MySQL](./mysql/_index.md#service-timezone)
+* [PostgreSQL](./postgresql.md#service-timezone)
 
-Once a service is running and exposed as a relationship, its appropriate credentials (host name, username if appropriate, etc.) will be exposed through the `PLATFORM_RELATIONSHIPS` environment variable.  The structure of each is documented on the appropriate service's page, along with sample code for how to connect to it from your application. Note that different applications manage configuration differently so the exact code will vary from one application to another.
+## Connect to a service
 
-Be aware that the keys in the `PLATFORM_RELATIONSHIPS` structure are fixed but the values they hold may change on any deployment or restart.  Never hard-code connection credentials for a service into your application.  You should re-check the environment variable every time your script or application starts.
+Once a service is running and exposed as a relationship,
+its appropriate credentials (such as the host, username, and password) are available through the `PLATFORM_RELATIONSHIPS` environment variable.
+The structure of each is documented on the appropriate service's page along with sample code for how to connect to it from your application.
 
-Access to the database or other services is only available from within the cluster.  For security reasons, they cannot be accessed directly.  However, they can be accessed over an SSH tunnel.  There are two ways to do so.  (The example here uses MariaDB but the process is largely identical for any service.)
+The keys in the `PLATFORM_RELATIONSHIPS` variable are fixed, but the values may change on deployment or restart.
+So you should check the environment variable every time your script or app starts.
 
-### Obtaining service credentials
+Access to the database or other services is only available from within the cluster.
```

Comment:
```suggestion
Access to the database or other services is only available from within the app.
```
I'm not sure what this meant before, sounds like it was meant for dedicated clusters?

Reply:
My understanding was it was to cover multi-app projects and such. But I agree the wording could be clearer.